fdbserver/DataDistribution.actor.cpp

```diff
@@ -1335,6 +1343,140 @@ ACTOR Future<Void> bulkLoadingCore(Reference<DataDistributor> self, Future<Void>
 	}
 }
 
+ACTOR Future<Void> doBulkDumpTask(Reference<DataDistributor> self,
```

Comment:
Whats this method doing? Trying to move a bulkDumpState along?

Reply:
This is an actor spawned by DD dedicated to listen on a SS bulkdump task.